enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/types.ts

```diff
@@ -4,21 +4,20 @@ import type { LoadQuestionHookResult } from "embedding-sdk/hooks/private/use-loa
 import type { SdkCollectionId } from "embedding-sdk/types/collection";
 import type { MetabasePluginsConfig } from "embedding-sdk/types/plugins";
 import type {
-  EntityTypeFilterKeys,
   LoadSdkQuestionParams,
   MetabaseQuestion,
   SdkQuestionId,
   SqlParameterValues,
 } from "embedding-sdk/types/question";
-import type { NotebookProps as QBNotebookProps } from "metabase/querying/notebook/components/Notebook";
+import type { EmbeddingEntityType } from "metabase/embedding-sdk/store";
 import type { Mode } from "metabase/visualizations/click-actions/Mode";
 import type Question from "metabase-lib/v1/Question";
 
 type InteractiveQuestionConfig = {
   /**
    * An array that specifies which entity types are available in the data picker
    */
-  entityTypeFilter?: EntityTypeFilterKeys[];
+  entityTypes?: EmbeddingEntityType[];
```

Comment:
@sanex3339 how do I fix the documentation here? There are 2 changes.
1. `entityTypeFilter` is removed and replaced by `entityTypes`
2. `entityTypes` has a slightly different type. It's a `"model" | "table"` rather than `"model" | "table" | "question" | "metric"`